### ISSUE TITLE
CI is failing

### ISSUE THREAD

**Issue by corneliusroemer** [bioconda/bioconda-recipes#40062]
It seems that CI is failing for all new recipes

**Comment by johnlees** [comment]
Yes, seems like timeout is being reached when installing bioconda-utils

**Comment by corneliusroemer** [comment]
Now it's fine again?

**Comment by johnlees** [comment]
Still failing for some at least

**Comment by johnlees** [comment]
(just on the linting!)

**Comment by corneliusroemer** [comment]
See https://github.com/bioconda/bioconda-recipes/issues/39986 https://github.com/bioconda/bioconda-recipes/issues/39999
